flink-python/src/main/java/org/apache/flink/table/functions/python/BeamTypeUtils.java

```diff
@@ -0,0 +1,548 @@
+		return logicalType.accept(new LogicalTypeToProtoTypeConverter());
+	}
+
+	private static class LogicalTypeToCoderConverter implements LogicalTypeVisitor<Coder> {
+
+		@Override
+		public Coder visit(CharType charType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(VarCharType varCharType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(BooleanType booleanType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(BinaryType binaryType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(VarBinaryType varBinaryType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(DecimalType decimalType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(TinyIntType tinyIntType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(SmallIntType smallIntType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(IntType intType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(BigIntType bigIntType) {
+			return BigIntCoder.INSTANCE;
+		}
+
+		@Override
+		public Coder visit(FloatType floatType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(DoubleType doubleType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(DateType dateType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(TimeType timeType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(TimestampType timestampType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(ZonedTimestampType zonedTimestampType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(LocalZonedTimestampType localZonedTimestampType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(YearMonthIntervalType yearMonthIntervalType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(DayTimeIntervalType dayTimeIntervalType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(ArrayType arrayType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(MultisetType multisetType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(MapType mapType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(RowType rowType) {
+			final Coder[] fieldCoders = rowType.getFields()
+				.stream()
+				.map(f -> f.getType().accept(this))
+				.toArray(Coder[]::new);
+			return new RowCoder(fieldCoders);
+		}
+
+		@Override
+		public Coder visit(DistinctType distinctType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(StructuredType structuredType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(NullType nullType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(AnyType<?> anyType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(SymbolType<?> symbolType) {
+			return null;
+		}
+
+		@Override
+		public Coder visit(LogicalType other) {
+			return null;
+		}
+	}
+
+	private static class LogicalTypeToBlinkCoderConverter implements LogicalTypeVisitor<Coder> {
```

Comment:
Make this class extends from the `LogicalTypeToCoderConverter`? Most parts of the two classes are same.